_posts/2019-03-07-strimzi-0.11.0-release.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+title:  "Strimzi 0.11.0 and 0.11.1 released!"
+date: 2018-03-07
+author: paolo_patierno
+---
+
+We are delighted to announce the new Strimzi 0.11.0 and 0.11.1 releases with some new features!
+
+<!--more-->
+
+# JBOD storage support for Kafka brokers
+
+Other than supporting the ephemeral and persistent storage, now Strimzi supports JBOD (Just a Bunch of Disks) storage for Kafka brokers (but not for Zookeeper nodes).
+
+The JBOD data storage configuration provides a way for having multiple disks combined to create a single logical disk, or volume.
+JBOD is one approach to providing increased data storage for Kafka brokers.
+It can also improve performance.
+A JBOD configuration is described by one or more volumes, each of which can be either ephemeral or persistent.
+
+To use JBOD with Strimzi, the storage `type` must be set to the new `jbod` value.
+Then the `volumes` property allows you to describe the disks that make up your JBOD storage array or configuration.
+
+Here's an example snippet with a JBOD configuration.
+
+```yaml
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    ...
+    storage:
+      type: jbod
+      volumes:
+      - id: 0
+        type: persistent-claim
+        size: 100Gi
+        deleteClaim: false
+      - id: 1
+        type: persistent-claim
+        size: 100Gi
+        deleteClaim: false
+```
+
+The Cluster Operator will configure the `log.dirs` Kafka parameter accordingly, in order to store messages logs on the different volumes.
```

Comment:
In this whole section you've used the terms disk and volume more or less interchangeably, but you not mentioned "Persistent Volumes" or PVCs. I think it would be better to tighten up the terminology, otherwise you risk confusing readers.